tests/test_observatory.py

```diff
@@ -217,3 +220,38 @@ def test_observatories_registered():
 
 def test_gbt_registered():
     get_observatory("gbt")
+
+
+def test_is_gbt_still_ok():
```

Comment:
The pytest plugins pytest-randomize and pytest-reverse could be handy to make this a tougher check (give it a chance to get run after any test that might cause a problem) - not that we should add them to `requirements_dev` but just for your own reassurance. Or if all the tests that might mess up the GBT are in one file, they normally get run in order, so this could be at the end.